provision-contest/ansible/handlers.yml

```diff
@@ -4,3 +4,18 @@
   command: "{{ DJ_DIR }}/webapp/bin/console cache:clear"
   become: true
   become_user: domjudge
+
+- name: rebuild domjudge
```

Comment:
It might not be needed yet here (this is not the full list of changes) but my idea was to detect the following cases:
- git shows we're not on the latest commit
- A file is missing

and than trigger the rebuild of domjudge, when I have time again I'll see if its doable to remove this part from the PR and make it 1 PR (for easier review)